install_config/aggregate_logging.adoc

```diff
@@ -111,72 +121,70 @@ API
 ----
 ====
 +
-Then, ensure the *logging-deployer* can edit the service. The following example
-uses the `logging` project. Be sure to use the project created above in the
-following example:
+Then, enable the *logging-deployer* service account to edit the project
+contents. By default service accounts are not allowed to do this.
+The following example uses the `logging` project. Be sure to use the
+project created above in the following example:
 +
 ====
 ----
 $ oc policy add-role-to-user edit \
-          system:serviceaccount:logging:logging-deployer
+            system:serviceaccount:logging:logging-deployer
+----
+====
+
```

Comment:
It seemed prudent to put the next section about the fluentd SA here, even though it's not created yet at this point. It gets all the security stuff out of the way up front instead of reintroducing it later in the flow.